package.json

```diff
@@ -10,6 +10,6 @@
   "devDependencies": {
     "tslint": "^4.4.2",
     "tslint-microsoft-contrib": "^4.0.0",
-    "powerbi-visuals-tools": "1.7.1"
+    "powerbi-visuals-tools": "1.10.0"
```

Comment:
As you use powerbi-visuals-utils-dataview-tuils you would need to add `"powerbi-visuals-utils-dataviewutils": "1.2.0"` as well.